arcgis-ios-sdk-samples/Layers/RGB renderer/RGB Renderer Cells.swift

```diff
@@ -0,0 +1,48 @@
+//
+// Copyright 2016 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+
+class RGBRendererInputCell: UITableViewCell {
+    
+    @IBOutlet weak var leadingLabel: UILabel!
+    @IBOutlet weak var textField: UITextField!
+    
+    let numberFormatter = NumberFormatter()
```

Comment:
There is no reason to give each cell its own number formatter. Formatters are expensive to create, so creating one per cell makes loading the table view necessarily expensive. In addition, it is the data source, not the cell itself, that should be responsible for configuring the cell's content. 

Please update the cell in the table view's data source instead. It is fine to have convenience methods like this on the data source to configure different kinds of cells. And by doing so, the data source can reuse a single number formatter for all cells.

Reply:
I disagree that this is a problem but I'll change it.